guides/installing_esphome.rst

```diff
@@ -49,19 +49,37 @@ with the following:
 Mac
 ---
 
-There are no tested installation instructions for Mac. ESPHome does support
-Mac & will run with no problem.
+ESPHome supports macOS and will run without issues.
 
-Contributions are welcome!
+An easy way for installation is via `Homebrew <https://brew.sh/>`_:
 
-The process will likely be similar to Windows. You can install Python from the
-official site, and then install ESPHome with ``pip3 install esphome``.  You can
-then test that things are properly installed with the following:
+.. code-block:: console
+
+    $ brew install esphome
+
+Verify the installation:
 
 .. code-block:: console
 
     $ esphome version
-    Version: 2021.12.3
+    Version: 2024.12.0 
+
+If you encounter any issues with Homebrew installation, please check the
+`ESPHome Homebrew Formula <https://formulae.brew.sh/formula/esphome>`_ page
+for additional information.
+
+Note that Homebrew may not always provide the latest version immediately. 
+For the latest version, use the pip installation. This may be more difficult to set up
+and may need additional dependencies and path settings.
+
+Install Python from the official site. Then, install ESPHome with 
```

Comment:
Expand this somewhat and make the code block level not inline, so make it stand out better. The Linux example below is an example to follow. Add a "Pip installation" sub-heading to make it easy to identify.